File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.py
class Solution:
    def removeAnagrams(self, words: List[str]) -> List[str]:
        res = []
        last = words[0]
        words[0] = "".join(sorted(words[0]))
        cur = ""
        for i in range(1, len(words)):
            cur = words[i]
            words[i] = "".join(sorted(words[i])) 
            if words[i] != words[i - 1]:
                res.append(last)
                last = cur
        res.append(last)
        return res

File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.cpp
class Solution {
public:
    vector<string> removeAnagrams(vector<string>& words) {
        vector<string> res;
        string last = words[0];
        sort(words[0].begin(), words[0].end());
        string cur;
        for(int i = 1; i < words.size(); i++)
        {
            cur = words[i];
            sort(words[i].begin(), words[i].end());
            if(words[i] != words[i - 1])
            {
                res.push_back(last);
                last = cur;
            }
        }
        res.push_back(last);
        return res;
    }
private:
    
};